editors/code/src/debug.ts

```diff
@@ -91,8 +91,8 @@ async function getDebugConfiguration(
 
     if (!debugEngine) {
         await vscode.window.showErrorMessage(
-            `Install [CodeLLDB](https://marketplace.visualstudio.com/items?itemName=vadimcn.vscode-lldb)` +
-                ` or [MS C++ tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools) extension for debugging.`,
+            `Install [CodeLLDB](command:rust-analyzer.installExtensionLLDB "Install CodeLLDB")` +
+                ` or [MS C++ tools](command:rust-analyzer.installExtensionCpp "Install MS C++ tools") extension for debugging.`,
```

Comment:
What are the strings after the command name? Do they get passed to the command?

Also, we should probably change "MS C++ tools" to "C/C++ for Visual Studio Code" or something like that.

Reply:
The strings after the command name are the tooltips shown when the user hovers over the command link, see video below, as a regular comment, since review comments don't support media.

Reply:
Well, the tooltips were in the original code, just the links didn't install the extensions, they led to the Marketplace via a browser, so I didn't remove them.

The current state of the PR installs the selected extension upon clicking, I am pretty sure searching would work as well. 😀